R/distribution.R

```diff
@@ -267,7 +267,7 @@ Distribution <-   function(x,
     ErrorIfNotEnoughData(x, require.tidy = FALSE, require.notAllMissing = TRUE)
     if (length(x) == 1 && is.list(x) && NCOL(x[[1]]) > 1)
         x <- x[[1]]
-    if (!is.list(x) && is.array(x))
+    if (!is.list(x) && (is.array(x) || inherits(x, "QTable")))
```

Comment:
So this fixes the case when `x` is a vector?

Reply:
I think the case of a vector is the next if else when it checks for is.numeric. But this check fixes the case in the unit test, which is a summary table for a numeric variable